spec/features/admin/workshops_spec.rb

```diff
@@ -193,5 +193,29 @@
         expect(page).to have_content('The requested format is invalid: text/html')
       end
     end
+
+    context 'attendee CSV' do
+      it 'returns a CSV with all workshop attendees' do
+        workshop = Fabricate(:workshop)
+        visit admin_workshop_path(workshop)
+        click_on 'Pairing CSV'
+
+        expect(page.current_path).to eq(admin_workshop_path(workshop, format: 'csv'))
+        expect(page).to have_content(WorkshopPresenter::PAIRING_HEADINGS.join(','))
+        expect(page).to_not have_content('ORGANISER')
```

Comment:
should this be `not_to` rather than `to_not`? Guessing it is fine but haven't run locally and usually use `not_to`!

Reply:
That does read a bit better so happy to make the change. I'm just accustomed to using `to_not`!